src/rise-twitter.html

```diff
@@ -1,2 +1,2 @@
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700" media="all">
-<div class="twitter-component-template theme-dark"></div>
```

Comment:
@rodrigopavezi It's supposed to be dark (meant for dark backgrounds with light text). Did something change?

Reply:
It should be on dark. Thanks

Reply:
@jungeunyoon @sheadarlison sorry I was not seeing the text on a white background and I thought that was the problem because there is on the doc that it will be defaulted to light theme. Makes sense based on the template I saw where the text is white on a dark background. I was wondering too. cheers